bpfd/src/command.rs

```diff
@@ -234,10 +236,15 @@ impl ProgramData {
                     .await
                     .map_err(ParseError::BytecodePullFaiure)?;
 
+                if program_overrides.image_meta.section_name != section_name { 
```

Comment:
If sec_name is null use overrides otherwise always use the provided section name